scripts/smmr/create_members_ior.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
```

Comment:
I think this file is not used. You used `create_members_50.sh` instead. Could we remove it?